doc/manual/source/interaction/geoserver_ui.rst

```diff
@@ -80,86 +74,75 @@ You can configure a store by:
-pull-down, enter a **Repository Name**, a **Parent Directory** and click Save.
+To configure a new GeoGig repository that is backed by the filesystem, select **Directory** from the **Repository Type** pull-down, enter a **Repository Name**, a **Parent Directory** and click "Save":
 
 .. figure:: ../img/create-new-geogig-repo-directory.png
 
-You can enter the Parent Directoy manually, or select one from a directory chooser dialog by clicking the **Browse...**
-link:
+You can enter the parent directory manually or select one from a directory chooser dialog by clicking the **Browse...** link:
 
 .. figure:: ../img/create-new-geogig-repo-directory-chooser.png
 
 .. _configure-new-postgres-repo:
 
-Configuring a new PostgreSQL backed GeoGig Repository
+Configuring a new PostgreSQL-backed GeoGig repository
 -----------------------------------------------------
 
-To configure a new GeoGig repository that is backed by a PostgreSQL database, select **PostgreSQL** from the
-**Repository Type** pull-down, enter the relevant database connection parameters and click Save.
+To configure a new GeoGig repository that is backed by a PostgreSQL database, select **PostgreSQL** from the **Repository Type** pull-down, enter the relevant database connection parameters and click "Save".
 
 .. figure:: ../img/create-new-geogig-repo-postgres.png
 
 .. _import-existing-repo:
 
-Importing an existing GeoGig Repository in GeoServer
+Importing an existing GeoGig repository in GeoServer
 ----------------------------------------------------
 
-You can create new GeoGig repositories through the :ref:`Create new GeoGig DataStore <configure-datastore-import-existing>`
-page or by naviagting to the `GeoGig Repositories` configuration page in the admin bar
+You can create new GeoGig repositories through the :ref:`Create new GeoGig datastore <configure-datastore-import-existing>` page or by navigating to the `GeoGig Repositories` configuration page in the admin bar
 
 .. figure:: ../img/configure-new-geogig-repo.png
 
 and selecting `Import an existing repository`
 
 .. figure:: ../img/import-existing-geogig-repo.png
 
-Just like creating new repositories, you have the option to import existing Directory backed repositories or PostgreSQL
-backed repositories. Select the **Repository Type** and choose/enter to repository location details:
+Just as when creating new repositories, you have the option to import existing Directory-backed repositories or PostgreSQL-backed repositories. Select the **Repository Type** and choose/enter the repository location details:
 
 .. figure:: ../img/import-existing-geogig-repo-directory.png
 
-   *Directory backed Repository configuration*
+   *Directory-backed Repository configuration*
 
 .. figure:: ../img/import-existing-geogig-repo-postgres.png
 
-   *PostgreSQL backed Repository configuration*
+   *PostgreSQL-backed Repository configuration*
 
-Cloning, Pushing and Pulling
+Cloning, Pushing, and Pulling
```

Comment:
The number of dashes `-` below needs to be the same (or greater) than the number of characters in the heading above.